cypress/e2e/room_board/createEditDeleteMultiColumnBoardInRoom.feature

```diff
@@ -0,0 +1,67 @@
+@regression_test
+@stable_test
+Feature: Room - Add, edit and delete board in room
+
+    As a teacher I want to add, edit and delete board in the room.
+
+    Scenario: Teacher add, edit, and delete board in the room, including pre-conditions
+        Given I am logged in as a '<teacher>' at '<namespace>'
+
+        # pre-condition: first teacher creating a new room
+        When I go to room overview
+        When I click on FAB to create new room
+        Then I see room creation page
+        When I enter the room name '<room_name>'
+        When I click on the button to save the room
+        Then I see the detail page of room '<room_name>'
+
+        # teacher creates a new multi column board
+        Given I am logged in as a '<teacher>' at '<namespace>'
+        When I go to room overview
+        When I go to room '<room_name>'
+        When I click on the add content button to see the creation button
+        Then I see the button to add board
+        When I click on the fab button to add a Board
+        Then I see the column board dialog box
```

Comment:
this line can be written as -> _Then I see the dialog box to select the Board type_